src/erasure-code/isa/ErasureCodeIsa.h

```diff
@@ -69,9 +69,11 @@ class ErasureCodeIsa : public ceph::ErasureCode {
             FLAG_EC_PLUGIN_ZERO_INPUT_ZERO_OUTPUT_OPTIMIZATION |
             FLAG_EC_PLUGIN_PARITY_DELTA_OPTIMIZATION;
 
-    if (technique == "reed_sol_van"sv ||
-        technique == "default"sv) {
-      flags |= FLAG_EC_PLUGIN_OPTIMIZED_SUPPORTED;
+    if (technique == "reed_sol_van"sv) {
```

Comment:
(just a note: I wanted to suggest to compare, now that 'technique' is a string, to ".."s and not ".."sv.
But for some reason - the binary produced is tighter on the 'sv'. Interesting.)